pkgs/tools/networking/remctl/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv, lib, autoreconfHook, fetchgit, libkrb5, libevent, openssl, perl
+, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "remctl";
+  version = "3.17";
+
+  src = fetchgit {
+    url = "git://git.eyrie.org/kerberos/remctl.git";
+    rev = "release/${version}";
+    sha256 = "1n7nwp2zqq806wbcnnamrsjfx250m4hfqqq94gz3fiqbgdrqs9pw";
+  };
+
+  outputs = [ "out" "perlsrc" "pythonsrc" ];
+
+  postPatch = ''
+    # Fix references to /usr/bin/perl in tests.
+    patchShebangs tests
+    # Disable acl/localgroup test that does not work in sandbox.
+    sed -i '/server\/acl\/localgroup/d' tests/TESTS
+  '';
+
+  nativeBuildInputs = [ autoreconfHook perl pkgconfig ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook perl pkg-config ];
```
pkgconfig is deprecated and ofborg should error about this.